iron-dropdown.html

```diff
@@ -434,46 +396,77 @@
         },
 
         /**
-         * Prepares the dropdown for opening by updating measured layout
-         * values.
+         * Updates the overlay position based on configured horizontal
+         * and vertical alignment.
          */
-        _prepareDropdown: function() {
-          this.sizingTarget = this.containedElement || this.sizingTarget;
-          this._updateAnimationConfig();
-          this._updateOverlayPosition();
+        _updateOverlayPosition: function() {
+          if (this.isAttached) {
+            // This triggers iron-resize, and iron-overlay-behavior will call refit if needed.
+            this.notifyResize();
+          }
         },
 
         /**
-         * Updates the overlay position based on configured horizontal
-         * and vertical alignment, and re-memoizes these values for the sake
-         * of behavior in `IronFitBehavior`.
+         * Positions the overlay based on configured horizontal
+         * and vertical alignment.
          */
-        _updateOverlayPosition: function() {
-          this._positionRectMemo = null;
-
-          if (!this.positionTarget) {
-            return;
-          }
+        center: function() {
+          var hAlign = this._localeHorizontalAlign;
+          var vAlign = this.verticalAlign;
+          // handle parent stacking contexts
+          this.style[hAlign] = this.style[vAlign] = '0px';
+          var rect = this.getBoundingClientRect();
```

Comment:
Aren't you afraid of doing this every scroll if outside scroll is allowed? Seems expensive.. :fearful: